### ISSUE TITLE
PNOR: Dynamically create systemd mount units

### ISSUE THREAD

**Issue by anoo1** [openbmc/openbmc#1848]
Reference: https://gerrit.openbmc-project.xyz/#/c/4794/
```
I think this works as a tactical means.
I think we should look at mounting these volumes from units that are created and destroyed as the volumes are created and populated.  They would depend on a unit that activates the ubi from the mtd device, and would be stored in the persistent rw volume, or an additional one in the pnor ubi as needed.
This would then become dynamically adding the units in the rw file systemd, or as a unit generator not sure.
```

Instead of auto-mounting the ubi volumes via a script, dynamically create unit files to mount the objects as versions get created.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/5423> PNOR: Dynamically create systemd mount units
Resolves  #1848 PNOR: Dynamically create systemd mount units

**Comment by saqibkh** [comment]
- [ ]  Create systemd mount files during activation.
- [ ]  Enable the systemd mount files to activate automatically during reboot
